src/styles/variables/custom-variables.scss

```diff
@@ -8,3 +8,12 @@ $secondary: #cccccc;
 $charcoal: #3a4750;
 $white: #ffffff;
 $winter-sky: #ff006e;
+
+// App gradient
+
+$whinter-sky-gradient: linear-gradient(
+    to bottom,
+    #303841 0%,
+    #303841 93%,
+    #ff006e 100%
+);
```

Comment:
@Andresws12 I think that is better use the name of the variables to use the colors in the gradient.